test_runner/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -38,10 +38,15 @@ object GcAndroidTestMatrix {
         obbFiles: Map<String, String>
     ): Testing.Projects.TestMatrices.Create {
         // https://github.com/bootstraponline/studio-google-cloud-testing/blob/203ed2890c27a8078cd1b8f7ae12cf77527f426b/firebase-testing/src/com/google/gct/testing/launcher/CloudTestsLauncher.java#L120
-        val clientInfo = ClientInfo()
-            .setName("Flank")
-            .setClientInfoDetails(args.clientDetails?.toClientInfoDetailList())
-
+        val clientInfo = if (androidTestConfig is AndroidTestConfig.Instrumentation && androidTestConfig.clientDetails.isNotEmpty()) {
+                            ClientInfo()
```

Comment:
Strange formatting, do you have ktlint installed?

Reply:
@asadsalman or you can just run`./gradlew formatKotlin` before commit